views/partials/header.ejs

```diff
@@ -328,7 +328,7 @@
 
 
 		<!-- MY NOTIFICATION REQUESTS - does NOT need to reload -->
-		<a id="danger-alert-box-button" class="inactive-window">
+		<a id="danger-alert-box-button" class="inactive-window" href="https://www.proavalon.com/lobby">
```

Comment:
Could you instead of `href="https://www.proavalon.com/lobby"`, do something like `href="/lobby"`? This way if someone else forks this and runs it locally, it will redirect it to the address the browser is currently on... Thanks! (pls test it, I don't remember the syntax off the top of my head)